src/code/z_sram_NES.c

```diff
@@ -1490,14 +1487,14 @@ void func_801457CC(GameState* gameState, SramContext* sramCtx) {
                     fileSelect->rupees[sp76] = gSaveContext.save.saveInfo.playerData.rupees;
                     fileSelect->walletUpgrades[sp76] = CUR_UPG_VALUE(UPG_WALLET);
 
-                    for (sp7A = 0, phi_a0 = 0; sp7A < 24; sp7A++) {
-                        if (gSaveContext.save.saveInfo.inventory.items[sp7A + 24] != 0xFF) {
-                            phi_a0++;
+                    for (sp7A = 0, maskCount = 0; sp7A < 24; sp7A++) {
+                        if (gSaveContext.save.saveInfo.inventory.items[sp7A + 24] == ITEM_NONE) {
```

Comment:
What do you think about this? I'm not totally convinced by it
```suggestion
                        if (gSaveContext.save.saveInfo.inventory.items[sp7A + SLOT_MASK_POSTMAN] == ITEM_NONE) {
```

This loop seems to just count how many masks the player has. I guess we could introduce stuff like `SLOT_MASK_MIN` and `SLOT_MASK_MAX`, but that is outside of the scope of this PR...

Feel free to ignore this whole suggestion lol, just throwing some thoughts.

Reply:
A macro already exists for this an was used in kaleido: `ITEM_NUM_SLOTS`.

I've gone ahead and used it here and in other places